File: Python Basics/exercises/shopping.py
budget = float(input())
pcs_videocard = int(input())
pcs_cpu = int(input())
pcs_ram = int(input())

videocard_price = 250
videocard_total_price = pcs_videocard * videocard_price
cpu_total_price = videocard_total_price * 0.35 * pcs_cpu
ram_total_price = videocard_total_price * 0.1 * pcs_ram

price = videocard_total_price + cpu_total_price + ram_total_price
if pcs_videocard > pcs_cpu:
    price = price - price * 0.15

difference = abs(budget - price)

if budget >= price:
    print(f"You have {difference:.2f} leva left!")
else:
    print(f"Not enough money! You need {difference:.2f} leva more!")

File: JS/Conditional Statements/Shopping.js
function calcMaterials(input){
    let budget = Number(input[0]);
    let numberOVideoCards = Number(input[1]);
    let numberOfProcessors = Number(input[2]);
    let numberOfRAM = Number(input[3]);
    let videoCards = 250 * numberOVideoCards;
    let processors = videoCards * 0.35 * numberOfProcessors;
    let ram = videoCards * 0.1 * numberOfRAM;
    let total = videoCards + processors + ram;
    if (numberOVideoCards > numberOfProcessors){
        total = total - total * 0.15;
    }
    let difference = Math.abs(budget - total);
    if (total > budget){
        console.log(`Not enough money! You need ${difference.toFixed(2)} leva more!`);
    }
    else{
        console.log(`You have ${difference.toFixed(2)} leva left!`);
    }
}

// calcMaterials(["920.45", "3", "1", "1"])